modules/batch/EED_Batch.module.php

```diff
@@ -93,6 +103,21 @@ public static function set_hooks_admin()
         add_action('wp_ajax_nopriv_espresso_batch_cleanup', array(self::instance(), 'batch_cleanup'));
     }
 
+    /**
+     * @since $VID:$
+     * @return LoaderInterface
+     * @throws InvalidArgumentException
+     * @throws InvalidDataTypeException
+     * @throws InvalidInterfaceException
+     */
+    protected function getLoader()
```

Comment:
Can't override `EED_Module::__construct()` (because it's `final`) to inject this, so just using this getter instead.

Reply:
ya that was my bad

Reply:
...although thinking about it, I suppose we could change that and it would be backward compatible, right? WDYT?

Reply:
ya nothing would break if the `final` declaration was removed, but modifying the constructor can be tricky